client/src/redux/sagas/datafiles.sagas.js

```diff
@@ -879,86 +880,72 @@ export function* emptyFile(system, path) {
   return 'SUCCESS';
 }
 
-export const getLatestApp = async (name) => {
-  const res = await fetchUtil({
-    url: '/api/workspace/apps',
-    params: {
-      publicOnly: true,
-      name,
-    },
-  });
-  const apps = res.response.appListing;
-  const latestApp = apps
-    .filter((app) => app.id.includes(name))
-    .reduce(
-      (latest, app) => {
-        if (app.version > latest.version) {
-          return app;
-        }
-        if (app.version < latest.version) {
-          return latest;
-        }
-        // Same version of app
-        if (app.revision >= latest.revision) {
-          return app;
-        }
-        return latest;
-      },
-      { revision: null, version: null }
-    );
-  return latestApp.id;
-};
-
 const getExtractParams = (file, latestExtract) => {
-  const inputFile = `agave://${file.system}${file.path}`;
-  const archivePath = `agave://${file.system}${file.path.substring(
-    0,
-    file.path.lastIndexOf('/') + 1
-  )}`;
+  const inputFile = encodeURI(`tapis://${file.system}/${file.path}`);
+  const archivePath = `${file.path.slice(0, -file.name.length)}`;
   return JSON.stringify({
-    allocation: 'FORK',
-    appId: latestExtract,
-    archive: true,
-    archivePath,
-    inputs: {
-      inputFile,
+    job: {
+      fileInputs: [
+        {
+          name: 'Input File',
+          sourceUrl: inputFile,
+        },
+      ],
+      name: `${latestExtract.definition.id}-${
+        latestExtract.definition.version
+      }_${new Date().toISOString().split('.')[0]}`,
+      archiveSystemId: file.system,
+      archiveSystemDir: archivePath,
+      archiveOnAppError: false,
+      appId: latestExtract.definition.id,
+      appVersion: latestExtract.definition.version,
+      parameterSet: {
+        appArgs: [],
+        schedulerOptions: [
+          {
+            name: 'TACC Allocation',
+            description:
+              'The TACC allocation associated with this job execution',
+            include: true,
+            arg: '-A TACC-ACI',
```

Comment:
this is related to the cannot-use-FORK-yet issue, right?  Do we need a TODOv3 listed here and/or follow on ticket?  and do we want to pick an allocation from the user's allocations instead of always TACC-ACI?

Reply:
Oops good catch, that should be a generic allocation. Yeah I'll add a follow on ticket for a FORK compress/extract, but it may turn out all our apps need to run through Slurm in order to get singularity working on some of these hosts

Reply:
@nathanfranklin Added an allocation selector